src/AqBanking/Account.php

```diff
@@ -14,22 +14,15 @@ class Account implements AccountInterface, Arrayable
      */
     private $accountNumber;
 
-    /**
-     * @var string
-     */
-    private $accountHolderName;
-
     /**
      * @param BankCode $bankCode
      * @param string $accountNumber
-     * @param string $accountHolderName
      * @return \AqBanking\Account
      */
-    public function __construct(BankCode $bankCode, $accountNumber, $accountHolderName = '')
+    public function __construct(BankCode $bankCode, $accountNumber)
     {
         $this->bankCode = $bankCode;
         $this->accountNumber = $accountNumber;
-        $this->accountHolderName = $accountHolderName;
```

Comment:
Why is the account holder name removed?

Reply:
Well it was optional and it is not being used by any of the aqbanking commands. Which makes it obsolete, if it is not used anywhere within the package. When the information is required by any consuming application it can be handled by itself - no need to pass it into the API of the package.